src/go/cmd/api_view.go

```diff
@@ -92,7 +92,7 @@ func CreateAPIView(pkgDir, outputDir string) error {
 	if outputDir == "." {
 		outputDir = ""
 	}
-	filename := fmt.Sprintf("%s%s.json", outputDir, pkg.p.Name)
+	filename := fmt.Sprintf("%s/%s.json", outputDir, pkg.p.Name)
```

Comment:
Could you do this instead: 
```
        if outputDir == "." {
		outputDir = ""
	} else if !strings.HasSuffix(outputDir, "/") {
		outputDir = fmt.Sprintf("%s/", outputDir)
	}
	filename := fmt.Sprintf("%s%s.json", outputDir, pkg.p.Name)
```

You can copy&paste this, I've already tested it.